### DISCUSSION TITLE
Python wrapper

### DISCUSSION THREAD

**Discussion by eadmaster** [rikyoz/bit7z#169] [Ideas]
provide an official python wrapper for bit7z.

I've only found this old project:
https://github.com/aver007/class_7zip_arch

[py7zr](https://github.com/miurahr/py7zr) is not a valid alternative because it only support 7z files.

**Comment by rikyoz** [comment]
Hi!
I'll definitely work on some Python bindings for a future version of bit7z.
Thanks for the suggestion!
